net-libs/mbedtls/mbedtls-2.25.0.recipe

```diff
@@ -0,0 +1,143 @@
+SUMMARY="Open source, portable, easy to use, readable and flexible SSL library"
+DESCRIPTION="mbed TLS (formerly known as PolarSSL) makes it trivially easy for \
+developers to include cryptographic and SSL/TLS capabilities in their (embedded) \
+products, facilitating this functionality with a minimal coding footprint."
+HOMEPAGE="https://tls.mbed.org/"
+COPYRIGHT="2006-2018 ARM Limited"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ARMmbed/mbedtls/archive/mbedtls-$portVersion.tar.gz"
+CHECKSUM_SHA256="ea2049c2dd4868693998d5a9780e198194be5aea1706ff4a9d4f882f18c0a101"
+SOURCE_DIR="mbedtls-mbedtls-$portVersion"
+PATCHES="mbedtls-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libcryptoVersion="3"
+libtlsVersion="12"
+libx509Version="0"
+
+PROVIDES="
+	mbedtls$secondaryArchSuffix = $portVersion
+	lib:libmbedcrypto$secondaryArchSuffix = $libcryptoVersion compat >= $libcryptoVersion
+	lib:libmbedtls$secondaryArchSuffix = $libtlsVersion compat >= $libtlsVersion
+	lib:libmbedx509$secondaryArchSuffix = $libx509Version compat >= $libx509Version
+	"
+if [ -z "$secondaryArchSuffix" ]; then
```

Comment:
Not at the laptop atm, but thinking of moving the binaries to a "tools" package, any thoughts? (if not I'll proceed) ;)